src/data.js

```diff
@@ -376,11 +435,19 @@ class Data extends QueryBuilder {
    *
    * @example {@lang javascript}
    * data.posts.delete(55)
+   * data.posts.delete([55, 56, 57])
```

Comment:
As far as I remember some of the things like this are possible to get also using one request and query e.g. (id _in [55, 56,57]). I'm not saying that batch here is wrong... @23doors am I right?